### PR TITLE
[rotelra1x] Migrate to bnd build

### PR DESCRIPTION
**Pull request by Hilbrand**
Automigrated with https://gist.github.com/Hilbrand/a7127d0fb7afd36124a78a248453ff03

### CODE REVIEW

File: bom/openhab-addons/pom.xml [hunk 1]
```diff
@@ -592,7 +592,7 @@
       <version>${project.version}</version>
     </dependency>
     <dependency>
-      <groupId>org.openhab.binding</groupId>
+      <groupId>org.openhab.addons.bundles</groupId>
```
Comment (martinvw) [inline bom/openhab-addons/pom.xml]: This looks unexpected?

Comment (martinvw) [reply bom/openhab-addons/pom.xml]: nevermind

**Comment by martinvw** [comment]
Do we have a way to check whether we did not forget to include any bindings in the BOM?

**Comment by Hilbrand** [comment]
@martinvw I have not checked or know if someone has checked if we forget to include any bindings in the bom. But I'm guessing it should not be to hard to script this.

**Comment by davidgraeff** [comment]
I have opened an issue about the idea of auto.generating the bom, if that's feasible

**Pull request merged**